### DISCUSSION TITLE
Problems building HyperHDR on macOS

### DISCUSSION THREAD

**Discussion by dattekachi** [awawa-dev/HyperHDR#720] [Q&A]
I built HyperHDR on the macOS version 13.6.1. This app can't be used on older versions of macOS
How to build HyperHDR to run on older versions of macOS. Thanks for any help.

**Comment by awawa-dev** [comment]
Please provide a screenshot. Does our HyperHDR v20beta1 release version work for you? What's your macOS device?

**Reply by dattekachi** [reply]
I built hyperhdr following the instructions: "github.com/awawa-dev/HyperHDR/wiki/Compiling-HyperHDR". I built it on macOS 13.6.1. When I tried to open hyperhdr that I built on macOS 13.0.1, it gave me an error message saying "you have macOS 13.0.1. The application requires macOS 13.6.1 or later".

**Reply by awawa-dev** [reply]
Ok but does our HyperHDR v20beta1 release version work for you? Currently we use Github Action macOS 12.x runner.

**Reply by awawa-dev** [reply]
As I compared https://github.com/awawa-dev/HyperHDR/blob/master/.github/workflows/push-master.yml with wiki maybe brew qt6 is the problem. Try qt5

**Reply by dattekachi** [reply]
I'll try, thank you very much.
